gnmi_server/client_subscribe.go

```diff
@@ -154,6 +153,7 @@ func (c *Client) Run(stream gnmipb.GNMI_SubscribeServer) (err error) {
 	log.V(1).Infof("Client %s running", c)
 	go c.recv(stream)
 	err = c.send(stream)
+
```

Comment:
minor:  are these lines added due to formatting? dont think they are needed?

Reply:
No they are not due `go fmt`, I'll get rid of these new lines.